src/hypermedia-constants.js

```diff
@@ -36,6 +36,7 @@ export const Rels = {
 	semesters: 'https://api.brightspace.com/rels/semesters',
 	sequence: 'https://api.brightspace.com/rels/sequence',
 	sorts: 'https://api.brightspace.com/rels/sorts',
+	specialization: 'https://api.brightspace.com/rels/specialization',
```

Comment:
I'd prefer if new ones stay in the file where they are used so we can start getting away from this file.